rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/LRACoordinatorRecovery2TestCase.java

```diff
@@ -140,6 +86,8 @@ public void testRecovery2(@ArquillianResource @OperateOnDeployment(COORDINATOR_D
         // start an LRA with a short timeout to validate that timed LRAs that time out when the coordinator is unavailable are cancelled
         URI shortLRA = lraClient.startLRA(null, "Short Timeout Recovery Test", SHORT_TIMEOUT, ChronoUnit.MILLIS);
 
+        doWait(1000);
```

Comment:
Thanks, I will tune the timeout ASAP.

This delay was introduced because the test was randomly failing to recover the Long & Short transactions. I am still investigating why this happened.